armi/reactor/grids.py

```diff
@@ -1410,6 +1410,8 @@ def _indicesAndEdgeFromRingAndPos(ring, position):
     pos = position - 1
 
     if ring == 0:
+        if pos != 0:
+            raise ValueError(f"Position in center ring must be 1, not {pos}")
```

Comment:
must be zero?

Reply:
well the user put in values into this method as 1-based but they get decremented to 0-based right above.